src/main/java/com/github/dockerjava/api/command/ListContainersCmd.java

```diff
@@ -38,16 +37,54 @@
     ListContainersCmd withBefore(String before);
 
     /**
+     * @param name
+     *            - Show only containers that has the container's name
+     */
+    ListContainersCmd withNameFilter(String... name);
+
+    /**
+     * @param id
+     *            - Show only containers that has the container's id
+     */
+    ListContainersCmd withIdFilter(String... id);
+
+    /**
+     * @param ancestor
+     *            - Show only containers created from an image or a descendant.
+     */
+    ListContainersCmd withAncestorFilter(String... ancestor);
+
+    /**
+     * @param volume
+     *            - Show only containers with volume name or mount point destination
+     */
+    ListContainersCmd withVolumeFilter(String... volume);
+
+    /**
+     * @param network
+     *            - Show only containers with network id or network name
+     */
+    ListContainersCmd withNetworkFilter(String... network);
+
+    /**
+     * @param exited
+     *            - Show only containers that exited with the passed exitcode.
+     */
+    ListContainersCmd withExitedFilter(Integer exited);
+
+    /**
+     * Use {@link #withExitedFilter(Integer)} instead
      * @param exitcode
      *            - Show only containers that exited with the passed exitcode.
      */
+    @Deprecated
     ListContainersCmd withExitcodeFilter(Integer exitcode);
```

Comment:
just remove it

Reply:
Did you mean that I should remove the withExitcodeFilter method completely

Reply:
yes, master in free state